deployments/dev/config/common.yaml

```diff
@@ -84,28 +84,16 @@ pangeo:
           name: custom-templates
           subPath: "pangeo-custom-jupyterhub-templates/extra-assets"
     auth:
-      github:
-        orgWhitelist:
-          - pangeo-data
-          - pydata
-          - dask
-          - xarray-dev
-          - jupyterhub
-          - informatics-lab
-      scopes:
-        - "read:org"
-        - "read:user"
+      auth0:
+        scope:
+          - openid
+          - profile
+          - email
```

Comment:
@consideRatio - Does this look correct? It's my attempt to translate https://github.com/jupyterhub/oauthenticator/blob/master/oauthenticator/auth0.py to yaml.